<?xml version="1.0" ?>

<launch>
  <!-- Arguments -->
  <arg name="cmd_vel_topic" default="/cmd_vel" />
  <arg name="odom_topic" default="odom" />

  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    <param name="base_global_planner" value="global_planner/GlobalPlanner" />
    <rosparam file="$(find neng)/cfg/turtlebot3/costmap_common_params_waffle.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find neng)/cfg/turtlebot3/costmap_common_params_waffle.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find neng)/cfg/turtlebot3/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find neng)/cfg/turtlebot3/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find neng)/cfg/turtlebot3/move_base_params.yaml" command="load" />
    <rosparam file="$(find neng)/cfg/turtlebot3/dwa_local_planner_params_waffle.yaml" command="load" />
    <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
  </node>
</launch>
